src/main/java/com/redhat/devtools/lsp4ij/LanguageServerWrapper.java

```diff
@@ -1233,4 +1266,21 @@ URI toUri(@NotNull VirtualFile file) {
         return FileUriSupport.getFileUri(file, getClientFeatures());
     }
 
+    /**
+     * Returns the language server wrapper's modification tracker.
+     *
+     * @return the modification tracker
+     */
+    @NotNull
+    ModificationTracker getModificationTracker() {
+        return modificationTracker;
+    }
+
+    /**
+     * Increments the language server wrapper's modification tracker.
+     */
+    @ApiStatus.Internal
+    public void incrementModificationCount() {
```

Comment:
And this is needed so that `LanguageServerWrapper` can bump the modification count when server status changes for wrappers that were not present when initial caching occurred.